bot/modules/community/actions.js

```diff
@@ -48,6 +48,20 @@ exports.onCommunityInfo = async ctx => {
   const userCount = await User.count({ default_community_id: commId });
   const orderCount = await getOrdersNDays(1, commId);
   const volume = await getVolumeNDays(1, commId);
+  
+  const creator = await User.findById(community.creator_id);
+  
+  let orderChannelsText = '';
+  if (community.order_channels.length === 1) {
+  	orderChannelsText = `${community.order_channels[0].name} (${community.order_channels[0].type})`;
+  } 
+  else if (community.order_channels.length === 2) {
```

Comment:
please give a look to the [coding standard docs](https://github.com/lnp2pBot/bot/blob/main/CONTRIBUTING.md#coding-standards) we are using to fix the indentation